src/main/java/dataAnalyzer/FormationResult.java

```diff
@@ -0,0 +1,33 @@
+package dataAnalyzer;
+
+import java.util.Calendar;
+
+public class FormationResult {
+    private Calendar startDate;
+    private Calendar endDate;
+    private long duration;
+    private String formationName;
```

Comment:
Why do we need to store the name of the formation? Also, if we do need to store it for some reason, then I would recommend making it an enum instead, because you can only have a finite amount of possible formations and using an enum would prevent bugs from typos.